src/js/test/locators.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+import createLocator from "./createLocator"
+
+export const jsonTypeConfigInput = createLocator("json_type_config_input")
+export const defaultModalButton = createLocator("default_modal_button")
+export const viewerResults = createLocator("viewer_results")
+export const ingestProgressBar = createLocator("ingest_progress_bar")
+export const toolbarExportButton = createLocator("toolbar_export_button")
```

Comment:
@mikesbrown here is another proposal for a new way to do integration test "locators". 

```js
// locators.js
export const myButton = createLocator("my_button")

// in react
<SomeButton {...myButton.props} />

// in an integration test
app.client.click(myButton.css)
```

What do you think of this pattern? I think it accounts for some of those good points you made in my previous PR, and has less friction that our current pattern.

Reply:
I think it's nice to split them up because one will rarely need to look at the createLocator function (after looking at it once). But I'll probably want to reference the locators more often. In general, it's helpful for me if a file only contains code that's relevant to my task at hand. I'm in favor of any structure that lowers the amount of focus I need to have when I open up a file. Hence, splitting them up.